src/d/actor/d_a_pw.cpp

```diff
@@ -26,7 +36,7 @@ static BOOL daPW_Draw(pw_class*) {
 }
 
 /* 000004D4-00000600       .text anm_init__FP8pw_classifUcfi */
-void anm_init(pw_class*, int, float, unsigned char, float, int) {
+void anm_init(pw_class*, int, float, J3DFrameCtrl::Attribute_e frameCtrl, float, int) {
```

Comment:
you can't change the types of function arguments from what they originally were as that changes the symbol name.

when I said this argument was the J3DFrameCtrl::Attribute_e enum, I just meant that it can be used at the callsites where `anm_init` is being called, not that `anm_init` itself needs to be changed to enforce the usage of the enum.

the name `frameCtrl` also isn't right, framectrl is the class that owns the attribute enum, not the enum itself.